sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/sarif/SarifParser10.java

```diff
@@ -185,7 +185,8 @@ private static boolean isSuppressed(JsonObject resultObj) {
   }
 
   private static String uriToAbsolutePath(String uri) {
-    return new File(URI.create(uri)).getAbsolutePath();
+    String uriEscaped = uri.replace("[", "%5B").replace("]", "%5D");
```

Comment:
Shouldn't it be reported as a bug to Microsoft?

Reply:
We'll look into it